modules/jupyter-widget/src/playground/create-deck.js

```diff
@@ -25,6 +25,18 @@ function extractClasses(library = {}) {
   return classesDict;
 }
 
+function extractFunctions(library = {}) {
```

Comment:
This is very similar to `extractClasses`. Perhaps create a shared method a supply the filter as a parameter?